modules/backend/main.js

```diff
@@ -0,0 +1,6 @@
+/******/ (() => { // webpackBootstrap
+/******/ 	"use strict";
+/******/ 	
+/******/ 	
+/******/ })()
+;
```

Comment:
?

Reply:
I've had that happen a couple of times if I've forgotten a `-p` argument for `mix:compile` when trying to compile a single module... I think it's some weird NPM flex. I'll make it an issue at some point and will clear it out of here.